TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/timeline/MainListTimeline.java

```diff
@@ -0,0 +1,261 @@
+package com.ybsystem.tweetmate.fragments.timeline;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Spinner;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.adapters.recycler.TweetRecyclerAdapter;
+import com.ybsystem.tweetmate.application.TweetMateApp;
+import com.ybsystem.tweetmate.models.entities.EntityArray;
+import com.ybsystem.tweetmate.models.entities.UserList;
+import com.ybsystem.tweetmate.models.entities.twitter.TwitterStatus;
+import com.ybsystem.tweetmate.databases.PrefSystem;
+import com.ybsystem.tweetmate.utils.CalcUtils;
+import com.ybsystem.tweetmate.utils.ExceptionUtils;
+import com.ybsystem.tweetmate.utils.ToastUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers;
+import io.reactivex.rxjava3.core.Observable;
+import io.reactivex.rxjava3.observers.DisposableObserver;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import twitter4j.Paging;
+import twitter4j.ResponseList;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+
+import static com.ybsystem.tweetmate.resources.ResString.*;
+
+public class MainListTimeline extends TimelineBase {
+
+    private long mSelectedListId;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate
+        View view = inflater.inflate(R.layout.fragment_timeline_list, container, false);
+
+        // Set view
+        setRecyclerView(view);
+        setRecyclerAdapter(new TweetRecyclerAdapter());
+        setUltraPullToRefresh(view);
+
+        // Fetch my list
+        fetchMyList(view);
+
+        return view;
+    }
+
+    protected void loadTweet(final boolean isPullLoad, final boolean isClickLoad) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 24 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS3776)